core/src/test/java/io/questdb/cutlass/line/tcp/LineTcpReceiverTest.java

```diff
@@ -28,10 +28,10 @@
 import io.questdb.cairo.pool.PoolListener;
```

Comment:
ILP with DROP and RENAME will probably not work and I don't want to fix it in this PR since it's already oversized by number of changes and time. We will need to test it after this and new WAL ILP PRs are merged.

Reply:
OK, addressing this separately makes sense to me.